docs/docs/pricing/README.md

```diff
@@ -96,6 +96,45 @@ If an event emitted by an event source triggers a single workflow, that will cou
 
 Your workflow's [memory settings](/workflows/settings/#memory) also impact the number of invocations you're charged for each workflow execution. [Read more here](#how-does-workflow-memory-affect-billable-invocations).
 
+#### Scenarios
+
+::: details Webhook triggered workflow
+
+*1* invocation is incurred per HTTP webhook. The HTTP endpoint is *not* considered a source.
+
+:::
+
+::: details Scheduled workflow
```

Comment:
Workflows run by the native Schedule / Custom Interval trigger technically aren't run by event sources, so you might remove this section since you already talk about Polling sources below.